webpack.config.babel.js

```diff
@@ -1,6 +1,7 @@
-const path = require("path");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const { name } = require('./package.json')
+import * as path from "path";
+import { CleanWebpackPlugin } from "clean-webpack-plugin";
+
+import { name } from "./package.json";
```

Comment:
Isn't it cool ?)

Reply:
Ok, but I quickly wrote them for npm setup